### DISCUSSION TITLE
[FEEDBACK] I don't want to see updates from my followers

### DISCUSSION THREAD

**Discussion by marlonrichert** [community/community#18557] [Other Feature Feedback, Questions, & Ideas]
If I wanted to see those, then I would be following _them._

Also, I really want to be able to opt out from updates like this:
> You're seeing this because you collaborated with X

Just because someone submitted an issue to my repo and I replied to them, doesn't mean I want to see their updates in my feed.

**Comment by Icecoldbeerr** [comment]
Hey great advise I appreciate the heads up

**Comment by MrKrishnaAgarwal** [comment]
Click **"Unsubscribe"**

**Reply by marlonrichert** [reply]
There is no such button there.
